maintainer/cuda_build.sh

```diff
@@ -0,0 +1,31 @@
+GIT_COMMIT=$(git rev-parse HEAD)
+URL=$(echo "https://gitlab.icp.uni-stuttgart.de/kai/espresso-build/-/jobs/${CI_JOB_ID}")
+STATUS="pending"
+curl "https://api.github.com/repos/kaiszuttor/espresso/statuses/$GIT_COMMIT?access_token=$GITHUB_TOKEN" \
+      -H "Content-Type: application/json" \
+      -X POST \
+      -d "{\"state\": \"$STATUS\", \"context\": \"ICP CUDA build\", \"target_url\": \"$URL\"}"
+
+whoami;
+nvidia-smi;
+cd ${CI_PROJECT_DIR};
+cp maintainer/configs/maxset.hpp myconfig.hpp;
```

Comment:
Can we make this script more generic like the Jenkins and Travis scripts where they can run different myconfigs?